doc/source/Reference/ScriptingReference/CommonOperations/index.md

```diff
@@ -96,6 +96,23 @@ end = script['frameRange']['end'].getValue()
 script.context().setFrame( frame )
 ```
 
+Metadata
+--------
+
+### Register a value for a plug/node
+
+```
+Gaffer.Metadata.registerValue( plug, "name", value )
+Gaffer.Metadata.registerValue( node, "name", value )
+```
```

Comment:
Should we document the register-by-type and persistent functionality? Or are those not common enough to go here?